Failure # 1 (occurred at 2022-04-05_19-43-53)
Traceback (most recent call last):
  File "/home/haoranliao/anaconda3/envs/tf2.7/lib/python3.9/site-packages/ray/tune/trial_runner.py", line 718, in _process_trial
    results = self.trial_executor.fetch_result(trial)
  File "/home/haoranliao/anaconda3/envs/tf2.7/lib/python3.9/site-packages/ray/tune/ray_trial_executor.py", line 688, in fetch_result
    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)
  File "/home/haoranliao/anaconda3/envs/tf2.7/lib/python3.9/site-packages/ray/_private/client_mode_hook.py", line 62, in wrapper
    return func(*args, **kwargs)
  File "/home/haoranliao/anaconda3/envs/tf2.7/lib/python3.9/site-packages/ray/worker.py", line 1494, in get
    raise value.as_instanceof_cause()
ray.exceptions.RayTaskError(BdbQuit): [36mray::UniTTN.train_buffered()[39m (pid=107588, ip=128.32.17.9)
  File "python/ray/_raylet.pyx", line 501, in ray._raylet.execute_task
  File "python/ray/_raylet.pyx", line 451, in ray._raylet.execute_task.function_executor
  File "/home/haoranliao/anaconda3/envs/tf2.7/lib/python3.9/site-packages/ray/_private/function_manager.py", line 563, in actor_method_executor
    return method(__ray_actor, *args, **kwargs)
  File "/home/haoranliao/anaconda3/envs/tf2.7/lib/python3.9/site-packages/ray/tune/trainable.py", line 173, in train_buffered
    result = self.train()
  File "/home/haoranliao/anaconda3/envs/tf2.7/lib/python3.9/site-packages/ray/tune/trainable.py", line 232, in train
    result = self.step()
  File "/tmp/pycharm_project_948/uni_ttn/tf2.7/non_distributed_tunning/model.py", line 223, in step
    self.model.run_epoch(batch_size, self.iteration, grad_accumulation=config['data']['grad_accumulation'])
  File "/tmp/pycharm_project_948/uni_ttn/tf2.7/non_distributed_tunning/model.py", line 223, in step
    self.model.run_epoch(batch_size, self.iteration, grad_accumulation=config['data']['grad_accumulation'])
  File "/home/haoranliao/anaconda3/envs/tf2.7/lib/python3.9/bdb.py", line 88, in trace_dispatch
    return self.dispatch_line(frame)
  File "/home/haoranliao/anaconda3/envs/tf2.7/lib/python3.9/bdb.py", line 113, in dispatch_line
    if self.quitting: raise BdbQuit
bdb.BdbQuit

